.github/workflows/ci.yaml

```
@@ -23,3 +23,9 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - run: make docker-build
+
+  lint-chart:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - make lint-chart
```

I'm sorry. I wrote it wrong.

```suggestion
    - run: make lint-chart
```

---

sorry, too🙇‍♂️